packages/cloudvision-connector/src/Wrpc.ts

```diff
@@ -233,18 +235,21 @@ export default class Wrpc {
   }
 
   /**
-   * Closes multiple streams in one `close` call.
+   * Closes multiple streams in one `close` call. If there are GET requests associated with any
```

Comment:
Would this comment not apply to the Connector level function? Here we just close any call (get or subscribe etc.)